src/scss/components/ec-btn/_ec-btn.scss

```diff
@@ -0,0 +1,148 @@
+// Here we state the default value of each color, if we change a brand
+// we can override the default value with something else
+$ec-btn--primary-color: $info !default;
+$ec-btn--primary-color-hover: $info-hover !default;
+$ec-btn--secondary-color: $level-4-interactive-elements !default;
+$ec-btn--secondary-color-hover: $level-3-body-and-headings !default;
+$ec-btn--success-color: $success !default;
+$ec-btn--success-color-hover: $success-hover !default;
+$ec-btn--error-color: $error !default;
+$ec-btn--error-color-hover: $error-hover !default;
+$ec-btn--disabled-color: $level-6-disabled-lines !default;
```

Comment:
There is lots of references to the `$white` in the code, shouldn't these be extracted here too? We might need to theme that too.